src/main/java/frc/shooter/ShooterSensors.java

```diff
@@ -0,0 +1,43 @@
+package frc.shooter;
+
+import frc.sensors.Limelight;
+import frc.shooter.MathHelpers;
+import frc.drives.DrivesSensors;
+
+public class ShooterSensors implements ShooterSensorsInterfeace 
+{
+	Limelight lightSensor; //Used for accessing member methods of the limelight.
+	DrivesSensors driveSensor; //Used for accessing member methods of the drive sensors.

```

Comment:
Don't need DriveSensors in here. This class is only for shooter sensors not drives